src/index.html

```diff
@@ -11,6 +11,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      method="get"
+      action="/"
+      data-qa="big"
+    >
+      <button
+        class="search-button"
+        type="submit"
+      >
+        <img
+          class="icon"
```

Comment:
Interesting solution 👍 
The button is actually redundant, there is an error on submit, so you can leave only icon image, but it's up to you